pkg/controller/controllerutil/controllerutil.go

```
@@ -181,6 +181,21 @@ func HasControllerReference(object metav1.Object) bool {
 	return false
 }
 
+// HasOwnerReference returns true if the owners list contains an owner reference
+// that matches the object's group, kind, and name.
+func HasOwnerReference(ownerRefs []metav1.OwnerReference, obj client.Object, scheme *runtime.Scheme) bool {
+	gvk, err := apiutil.GVKForObject(obj, scheme)
+	if err != nil {
+		return false
```

Can we return an error here, please?